diff --no-dereference -uNr orig/usr/share/X11/xkb/keycodes/macintosh patched/usr/share/X11/xkb/keycodes/macintosh
--- orig/usr/share/X11/xkb/keycodes/macintosh	2021-06-09 03:20:40.000000000 +0700
+++ patched/usr/share/X11/xkb/keycodes/macintosh	2021-11-02 11:05:50.519148313 +0700
@@ -36,6 +36,11 @@
    
 };
 
+xkb_keycodes "badmap" {
+    <TLDE> =  94;
+    <LSGT> =  49;
+};
+
 xkb_keycodes "old" {
 
     minimum= 8;
diff --no-dereference -uNr orig/usr/share/X11/xkb/rules/base patched/usr/share/X11/xkb/rules/base
--- orig/usr/share/X11/xkb/rules/base	2021-08-18 16:26:23.000000000 +0700
+++ patched/usr/share/X11/xkb/rules/base	2021-11-02 11:05:27.603157636 +0700
@@ -169,6 +169,10 @@
   $qwertz       =       +aliases(qwertz)
   *             =       +aliases(qwerty)
 
+! option    =   keycodes
+  apple:badmap  =   +macintosh(badmap)
+  apple:goodmap =   +macintosh(goodmap)
+
 ! option	=	keycodes
 
 ! model		layout	=	geometry
